javascript-operators-and-expressions/javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,46 @@
+// Question 6i
+const width = 10;
+const height = 10;
+const area = width * height;
+console.log('value of area:', area);
+console.log('typeof area:', typeof area);
+
+// Question 6ii
+const bill = 100;
+const payment = 200;
+const change = payment - bill;
+console.log('value of change:', change);
+console.log('typeof change:', typeof change);
+
+// Question 6iii
+const quizzes = 10;
+const midterm = 50;
+const final = 40;
+const grade = (quizzes + midterm + final) / 3;
+console.log('value of grade:', grade);
+console.log('typeof grade:', typeof grade);
+
+// Question 6iv
+const firstName = 'Roland';
+const lastName = 'Estropia Jr.';
+const fullName = firstName + ' ' + lastName;
+console.log('value of fullName:', fullName);
+console.log('typeof fullName:', typeof fullName);
+
+// Question 6v
+const pH = 10;
+const isAcidic = pH < 7;
+console.log('value of isAcidic:', isAcidic);
+console.log('typeof isAcidic:', typeof isAcidic);
+
+// Question 6vi
+const headCount = 10;
+const isSparta = headCount === 300;
+console.log('value of isSparta:', isSparta);
+console.log('typeof isSparta:', typeof isSparta);
+
+// Question 6vii
+let motto = fullName;
+motto = fullName + ' is the GOAT';
```

Comment:
You should be using the `+=` operator here

Reply:
![javascript-operators-and-expressions- 2](https://user-images.githubusercontent.com/66459527/236952585-2d62900e-65d2-4ed4-a1b4-06e6be0ac3e1.PNG)

Added (see picture)